src/helpers/blood-pressure-data-providers/survey-blood-pressure-data-provider.ts

```diff
@@ -1,11 +1,12 @@
 import MyDataHelps, { Guid, SurveyAnswer, SurveyAnswersPage, SurveyAnswersQuery } from "@careevolution/mydatahelps-js";
-import { format, isDate, parseISO, startOfDay, toDate } from "date-fns";
+import { parseISO, startOfDay } from "date-fns";
 
 export interface SurveyBloodPressureDataParameters {
     surveyName: string,
     dateResultIdentifier: string,
     systolicResultIdentifier: string,
-    diastolicResultIdentifier: string
+    diastolicResultIdentifier: string,
+    hideDeviceData?: boolean
```

Comment:
I think this should not be on the SurveyBloodPressureDataParameters since this is about device data; we should put this parameter on the main visualization.  

For simplicity I would probably include a parameter on the main BloodPressureVisualization that is a ```BloodPressureDeviceDataSource[]``` where ```type BloodPressureDeviceDataSource = 'AppleHealth' | 'GoogleFit'```

And then that gets passed into the data provider stuff so it knows what device data sources to include

Reply:
Addressed